docs/snippets/ov_hetero.cpp

```diff
@@ -46,9 +46,9 @@ auto compiled_model = core.compile_model(model, device);
         // GPU with fallback to CPU
         ov::device::priorities("GPU", "CPU"),
         // profiling is enabled only for GPU
-        ov::device::properties("GPU", ov::enable_profiling(true)),
+        ov::properties("GPU", ov::enable_profiling(true)),
```

Comment:
It looks like non-backward compatible change. Am I wrong?

Reply:
Yes you are right. It is needed to add aliases for backward compatebility.

Reply:
added backward compatible variable